src/main/java/BattleshipModel.java

```diff
@@ -64,6 +65,46 @@ public void placeShipAI(){
         generateShipLocation(ai,"Battleship",4);
         generateShipLocation(ai,"Carrier",5);
     }
+
+    //This could either go in a
+    public void placeShipAIEasy(){
+        //This could probably be shortened, but this should be fine.
+        Coordinate start = new Coordinate(3,3);
+        Coordinate end = new Coordinate(3,5);
+        Ship s = new Civillianship("Clipper",3,start,end,"vertical",true);
+        ai.addShip(s);
+
+        start = new Coordinate(5,3);
+        end = new Coordinate(5,5);
+        s = new Battleship("Submarine",3,start,end,"vertical",false);
+        ai.addShip(s);
+
+        start = new Coordinate(9,5);
+        end = new Coordinate(9,5);
+        s = new Civillianship("Dinghy",1,start,end,"vertical",true);
+        ai.addShip(s);
+
+        start = new Coordinate(2,6);
+        end = new Coordinate(5,6);
+        s = new Battleship("Battleship",4,start,end,"horizontal",true);
+        ai.addShip(s);
+
+        start = new Coordinate(1,1);
+        end = new Coordinate(5,1);
+        s = new Battleship("Carrier",5,start,end,"horizontal",true);
+        ai.addShip(s);
+    }
```

Comment:
It's long but I guess you can't get around that when you have to place so many ships statically so seems fine. Doesn't look like any of the ships have collisions.